optical/converter/utils.py

```diff
@@ -99,3 +101,82 @@ def copyfile(
         shutil.copyfile(filename, dest)
     except FileNotFoundError:
         pass
+
+
+def write_xml(
```

Comment:
@bharatkumarreddy , are you sure this `write_xml` is representing XML hierarchy correctly? can you attach converted XML sample. because all elements are `xml.Element` but I see for example `object` is a sub element of `annotation`

Reply:
I am unable to attach the xml format here but attaching the screenshot of xml

<img width="822" alt="Screenshot 2021-04-14 at 7 50 41 PM" src="https://user-images.githubusercontent.com/6673708/114726186-c6089200-9d5a-11eb-8eb5-2260eb8c04ff.png">